backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/MeetingService.cs

```diff
@@ -250,5 +281,20 @@ public async Task DeleteMeeting(long meetingId)
 
             await _context.SaveChangesAsync();
         }
+
+        private EmailDto GenerateNewMeetingEmailTemplate(Meeting meeting, string member)
+        {
+            return new EmailDto()
+            {
+                Subject = $"New Event: {meeting.Name} - {meeting.StartTime} - {meeting.LocationType}",
+                Body = $"Hi, {member}!\n\n" +
+                $"A new event has been scheduled.\n" +
+                $"Event Type: {meeting.LocationType}\n" +
+                $"Invitee: {meeting.Author.Name}\n" +
+                $"Invitee Email: {meeting.Author.Email}\n" +
+                $"Event Date/Time: {meeting.StartTime}\n\n" +
+                $"View event in Easymeets: {_applicationURL}{meeting.MeetingLink}"
+            };
+        }
```

Comment:
Create a separate method for parameters mapping, so we can reuse this mapping for other templates